authnz/src/main/java/io/stargate/auth/SourceAPI.java

```diff
@@ -17,11 +17,19 @@
  */
 package io.stargate.auth;
 
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
 public enum SourceAPI {
```

Comment:
I know it didn't exist before, but Javadoc here would be nice; could mention that value is passed along as custom metadata (for CQL) or headers (bridge/grpc).